templates/vue-demo-store/components/product/ProductAddToCart.vue

```diff
@@ -9,9 +9,17 @@ const { product } = toRefs(props);
 const { getErrorsCodes } = useCartNotification();
 const { t } = useI18n();
 const { addToCart, quantity } = useAddToCart(product);
+const {
+  addToCart: customizedProductAddToCart,
+  isActive: isCustomizedProductActive,
+} = useProductCustomizedProductConfigurator();
 
 const addToCartProxy = async () => {
-  await addToCart();
+  if (isCustomizedProductActive.value) {
+    await customizedProductAddToCart();
```

Comment:
Can we add this logic to the `addToCart`? `useAddToCart` composable should be more flexible

Reply:
I'm here to discuss about it because It's not clear yet where to put the example. Currently it's a part of the template only, and the composable itself is located here, not as a part of composables package. The reason is that 
the composables are designed to be more "core"-ish,  would be for a core-related features only, and the conditionally enabled feature like "custom product" would be easy to delete/change in the template level. WDYT?